glass/arraytools.py

```diff
@@ -11,12 +11,13 @@
     from collections.abc import Sequence
     from typing import Unpack
 
+    from jax.typing import ArrayLike
```

Comment:
Update: this should be `from jaxtyping import Array` to avoid having a dependency on jax (large dependency, does not work for pre-commit.ci servers).

Reply:
Okay thanks!